src/shared/codelens/localLambdaRunner.ts

```diff
@@ -310,3 +330,320 @@ export class LocalLambdaRunner {
         }
     }
 }
+
+// tslint:disable-next-line:max-line-length
+export async function run({baseBuildDir, channelLogger, codeDir, configuration, debugConfig, documentUri, handlerName, isDebug, localInvokeParams, manifestPath, runtime, samProcessInvoker, samTaskInvoker, workspaceUri}: {
```

Comment:
This argument list is out of control. Consider something like

```typescript
interface MakeInputTemplateOptions { ... }
function makeInputTemplate(options: MakeInputTemplateOptions) { ... }

interface ExecuteSamBuildOptions { ... }
function executeSamBuildOptions(options: ExecuteSamBuildOptions) { ... }

export async function run({
    ...
}: MakeInputTemplateOptions & ExecuteSamBuildOptions & {
    ...
}): Promise<void> { ... }
```

Reply:
True. Eliminated `run` since it's not used.
✔️